### PR TITLE
Fixes #1692 Improving Toggle Button

### LINKED ISSUE #1692: Improving Toggling Button  for Card
**I'm submitting a ...**
- [x] feature request

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes # #1692 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.

#### Changes proposed in this pull request:
When Toggle is off.
screenshot from 2018-09-14 09-22-40

When Toggle is on .
screenshot from 2018-09-14 09-22-43

@ParthS007  @gabru-md Please review it and suggest changes.

### CODE REVIEW

**Comment by sk9331657** [comment]
@ParthS007 @gabru-md Please review it.

**Comment by gabru-md** [comment]
I am not quite sure as if we need this change as of now. It was mentioned to make it the way we have it currently. Can you please leave this open and tag other devs at gitter so that they can also comment their opinion on the toggle button that you've proposed

File: frontend/app/templates/create-badges.hbs [hunk 1]
```diff
@@ -1,8 +1,12 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" style="margin-right:50px;">
+        
+
+<label class="switch" {{action 'togglePreview'}} onclick="document.getElementById('toggle').checked=!document.getElementById('toggle').checked" id="switch"   data-tooltip="Toggle Preview" data-position="left center" >
```
Comment (gabru-md) [inline frontend/app/templates/create-badges.hbs]: `onclick` is already being triggered when it is clicked in the `{{action 'togglePreview'}}` I suppose.
Can you please see to it and make changes ?

**Comment by sk9331657** [comment]
@abishekvashok  We can improve the toggling button like this.

ezgif com-optimize

File: frontend/app/templates/components/data-component/csv-component.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="field">
     <label>Upload CSV</label>
-    <input name="csv" onchange={{action "encodeCSV"}} type="file">
+    <input name="csv" onchange={{action "encodeCSV"}} type="file" accept=".csv">
```
Comment (ParthS007) [inline frontend/app/templates/components/data-component/csv-component.hbs]: PR related to this change already Merged.

File: frontend/app/templates/create-badges.hbs [hunk 3]
```diff
@@ -1,8 +1,12 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" style="margin-right:50px;">
```
Comment (ParthS007) [inline frontend/app/templates/create-badges.hbs]: Avoid Inline styling.

File: frontend/app/templates/create-badges.hbs [hunk 4]
```diff
@@ -1,8 +1,12 @@
...
+        
+
+<label class="switch" {{action 'togglePreview'}}  id="switch"   data-tooltip="Toggle Preview" data-position="left center" >
```
Comment (ParthS007) [inline frontend/app/templates/create-badges.hbs]: Don't use IDs for styling.

**Comment by ParthS007** [comment]
@sk9331657  Please update your PR.

File: frontend/app/templates/components/data-component/csv-component.hbs (hunk 2 above)
Comment (sk9331657) [reply frontend/app/templates/components/data-component/csv-component.hbs]: @ParthS007  It will overwrite that . Will it make any  issues??

**Comment by sk9331657** [comment]
@ParthS007  Done Changes.

File: frontend/app/templates/create-badges.hbs (hunk 4 above)
Comment (sk9331657) [reply frontend/app/templates/create-badges.hbs]: Ok

**Comment by sk9331657** [comment]
@ParthS007 @abishekvashok  @gabru-md  Please review this .

**Comment by sk9331657** [comment]
@ParthS007 Look at this one too.

File: frontend/app/templates/components/data-component/csv-component.hbs (hunk 2 above)
Comment (abishekvashok) [reply frontend/app/templates/components/data-component/csv-component.hbs]: I don't know why this is here

**Comment by abishekvashok** [comment]
Update your branch by pulling from the development branch and deploy your current work on top of it: 
`git pull --rebase origin development`
where origin is remote of this repo;

If you encounter any conflicts while you do so, or if the diff of the last commit doesn't look good, change the files and commit using the ammend parameter, so that the diff will be corrected for the last commit. It's done by: `git commit --amend`

**Pull request closed without merging**